maintainers/maintainer-list.nix

```diff
@@ -9087,6 +9087,12 @@
     githubId = 65870;
     name = "Сухарик";
   };
+  sumnerevans = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add sumnerevans``.

Reply:
I think I fixed the commit history.